src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -292,3 +293,13 @@
 
 (rf/reg-event-fx :wallet/make-partially-operable-accounts-fully-operable
  make-partially-operable-accounts-fully-operable)
+
+(rf/reg-event-fx :wallet/show-disable-testnet-mode-confirmation
+ (fn [_]
+   {:fx [[:dispatch
+          [:show-bottom-sheet
+           {:content (fn [] [testnet/view
+                             {:enable? false
+                              :blur?   true}])
+            :shell?  true
+            :theme   :dark}]]]}))
```

Comment:
I would imagine the bottom sheet confirmation should respect the current app theme, but if the user is on light mode we will show the dark variant. Seems a bit inconsistent because the other bottom sheets are being displayed with the light variant outside the settings screen.

Reply:
It works for now @mohsen-ghafouri 👍🏼 As a user of light mode in many occasions, I simply can't read text on dark backgrounds when outdoors, and I guess that's true for many users, or maybe I'm just getting old. So that's why it caught my attention seeing a bottom sheet in dark mode outside the settings.

Reply:
Yes, on iOS with blur looks okay. On Android without blur pitch black almost.

<img src="https://github.com/user-attachments/assets/a8ab40e2-68c1-479a-9437-5f4833b300b6" width="300" />

Reply:
Ah, didn't check that. will fix it. thanks for pointing it out 🙏

Reply:
It looks all shell? in android are showing with dark background regardless of selected theme, like share modal.

Reply:
Resolved, also added screenshots to the description